docs/source/usingmame/defaultkeys.rst

```diff
@@ -1,106 +1,476 @@
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+
+
+.. _default-computer-keys:
+
+Default Computer Keys
+=====================
+
+All the keys below are fully configurable in the user interface. This list shows the standard keyboard configuration.
+
+Note that controls can vary widely by computer type, so not all keys are shown here. See the "Input (this Machine)" section of MAME's
+configuration menu for details for the machine you are currently using.
+
+
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| Key                                | Action                                                                                                    |
++====================================+===========================================================================================================+
+| **Tab**                            | Toggles the configuration menu.                                                                           |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **Scroll Lock**                    | Default mapping for the **uimodekey**.                                                                    |
+|                                    |                                                                                                           |
+|                                    | | This key allows users to disable and enable the emulated keyboard in machines that require it.          |
+|                                    | | All emulations which require emulated keyboards will start in that mode and you can only access         |
+|                                    | | the internal UI (hitting TAB) by first hitting this key. You can change the initial status of the       |
+|                                    | | emulated keyboard as presented upon start by using **-ui_active**.                                      |
+|                                    |                                                                                                           |
+|                                    | FIXME: Add link back to -ui_active                                                                        |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **Alphanumeric Keys**              | These keys are mapped to their equivalents in the emulated machine by default.                            |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
```

Comment:
Tape controls probably belong in this section, and the UI paste control.

Reply:
Got those now as well.